assets/src/stories-editor/components/font-family-picker/index.js

```diff
@@ -63,6 +63,7 @@ function FontFamilyPicker( {
 						inputValue: inputValueTemplate,
 					}
 				}
+				ariaLabel={ __( 'Font Family', 'amp' ) }
```

Comment:
Isn't this redundant when the `BaseControl` already displays a label?

The bug seems to be that it displays the label for the element with ID `id`, but the `Autocomplete` component internally uses `${ id }__listbox` as the ID, and not just `id` alone.

So the correct fix would be passing `id={ `${ id }__listbox` }` to the `BaseControl` so that the label actually references an existing element.

Reply:
That solution didn't work ^ . I tried something similar in development.

This solution did work - https://github.com/ampproject/amp-wp/pull/3294/commits/8fee020a2503f13eedb5f32a8868bc648fd50a94

Reply:
I don't fully follow with this is trying to fix.

I understand that AXE is complaining, that the rendered `<input />` inside the autocomplete doesn't have a label (a `<label />` or an `aria-label`), right? And this fix ads a label (via `aria-label`), though it's kind of a duplicate.

But it does have a `<label />` **and** the label's `for` attribute correctly references the `id` of the `<input />`:
![label_for_input](https://user-images.githubusercontent.com/637548/65504397-2446f900-dec7-11e9-81d9-b0c868e69111.gif)

So I don't really follow neither the problem nor the fix?

Reply:
Oh, I see. The issue was with the `listbox` which, other than the `input`, didn't have a label associated with it.

If there's a visible label, then `aria-labelledby` should be used to refer to that label. 8fee020a2503f13eedb5f32a8868bc648fd50a94 did this.

Reply:
So to summarize this:

* The `input` field has "Font Family" as its label (due to the `for` attribute).
* The list of items (`listbox`) has "Type to search for fonts` as its label (due to  the `aria-labelledby`).

In my testing with VoiceOver the help text was not read aloud at all, but perhaps it's done by other screen readers.

In any case, even if this passes the AXE test it would be good to have some manual tests for this independent of this PR, once it's merged.

Reply:
The list box is NOT the input. It is the `<ul>` that contains the autocomplete options.

Reply:
So much is clear yes